mentors/elixir/j.json

```
@@ -14,5 +14,13 @@
     "link_url": "https://julioleitao.github.io",
     "avatar_url": "https://s.gravatar.com/avatar/bded7904d07d650f9a51fcdb7d9f2408?s=200",
     "bio": "Enjoying the Functional Programming paradigm to solve problems and build solutions with quality, maintainability, and great code design. Elixir is one of the best options on the FP world."
+  },
+  {
+    "github_username": "joaofcosta",
+    "name": "Dino",
+    "link_text": "Github",
```

Travis CI is stickler on capitalization/spelling of GitHub.
```suggestion
    "link_text": "GitHub",
```